style/testing/README.md

```diff
@@ -9,6 +9,7 @@ Testing
 * Use RSpec's [`expect` syntax].
 * Use RSpec's [`allow` syntax] for method stubs.
 * Use `not_to` instead of `to_not` in RSpec expectations.
+* Prefer to test the string over a I18n translation key.
```

Comment:
To use proper english and be more explicit, I'd suggest: 

```
Prefer testing the resulting string to calling I18n.
```

Reply:
👍  @3wille